src/graph_gui.cpp

```diff
@@ -186,6 +186,8 @@ struct BaseGraphWindow : Window {
 	byte month;
 	Year year;
 
+	bool draw_dates; ///< Should we draw months and years on the time axis?
```

Comment:
Maybe I missed something, but where is this variable set to true? I only found line 887 where it is set to false.
And shouldn't lines 502 and 527 also be updated to use `draw_dates`?

Reply:
You didn't miss anything, I missed a few things while cherry-picking. Thanks for the heads-up.